### ISSUE TITLE
Ruby/C++ Interop Flake timeout_on_sleeping_server

### ISSUE THREAD

**Issue by kpayson64** [grpc/grpc#7276]
```
Warning! PATH is not properly set up, '/usr/local/rvm/gems/ruby-2.1.8/bin' is not available,
         usually this is caused by shell initialization files - check them for 'PATH=...' entries,
         it might also help to re-add RVM to your dotfiles: 'rvm get stable --auto-dotfiles',
         to fix temporarily in this shell session run: 'rvm use ruby-2.1.8'.
Using /usr/local/rvm/gems/ruby-2.1.8
stdin: is not a tty
I0708 00:03:56.766125924       1 ev_epoll_linux.c:84]        epoll engine will be using signal: 36
D0708 00:03:56.766162041       1 ev_posix.c:106]             Using polling engine: epoll
src/ruby/pb/test/client.rb:83:in `assert': timeout_on_sleeping_server: status was wrong (AssertionError)
	from src/ruby/pb/test/client.rb:343:in `rescue in timeout_on_sleeping_server'
	from src/ruby/pb/test/client.rb:337:in `timeout_on_sleeping_server'
	from src/ruby/pb/test/client.rb:473:in `call'
	from src/ruby/pb/test/client.rb:473:in `main'
	from src/ruby/pb/test/client.rb:478:in `<main>'
I, [2016-07-08T00:03:56.951403 #1]  INFO -- : ... connecting securely to localhost:34451
W, [2016-07-08T00:03:57.953452 #1]  WARN -- : bidi: read-loop failed
W, [2016-07-08T00:03:57.953526 #1]  WARN -- : 13:{"created":"@1467936237.953122416","description":"RST_STREAM","file":"src/core/ext/transport/chttp2/transport/frame_rst_stream.c","file_line":107,"http2_error":2} (GRPC::BadStatus)
/var/local/git/grpc/src/ruby/lib/grpc/generic/active_call.rb:47:in `check_status'
/var/local/git/grpc/src/ruby/lib/grpc/generic/bidi_call.rb:212:in `block (2 levels) in start_read_loop'
/var/local/git/grpc/src/ruby/lib/grpc/generic/bidi_call.rb:200:in `loop'
/var/local/git/grpc/src/ruby/lib/grpc/generic/bidi_call.rb:200:in `block in start_read_loop'
```

**Comment by kpayson64** [comment]
https://grpc-testing.appspot.com/job/gRPC_interop_pull_requests/6184/testReport/junit/(root)/tests/cloud_to_cloud_ruby_c___server_timeout_on_sleeping_server/

This was on a python PR, I'm not sure if its been seen in master as well.

**Comment by stanley-cheung** [comment]
This is somewhat blocking PR submitted by external developers. There is an issue with Jenkins where hitting "Rebuild" will just fail immediately for those PRs. So there is no easy way to retry these flakes.

**Comment by murgatroid99** [comment]
This appears to no longer be happening. I think it was fixed by #7291. If we see it again, we can reopen this issue.
